mne/preprocessing/hfc.py

```diff
@@ -26,6 +28,10 @@ def apply_hfc(raw, order=1, exclude_bads=True, verbose=None):
     exclude_bads: bool
         Do not include bad channels in the projection, or alter their data
         (default: True).
+    pm: bool
```

Comment:
Rather than this API, you can instead do `accuracy='accurate' | 'normal' | 'point'` (I think?) to choose the coil_def accuracy to use when creating the coils. In theory `'point'` should get you back to the point mag (1-point) approximation

Reply:
This was my original plan, but to make it work you need to interfere with the [accuracy dictionary](https://github.com/mne-tools/mne-python/blob/6384a8901182272c48f3e72a10142ea75184f47f/mne/forward/_make_forward.py#L38), where the point setting appears to have been itentionally omitted to stop users selecting a point magnetometer setup. 

If everyone is happy to add point back into there that will simplify things.

Reply:
It's private so we have some flexibility. I think in whatever private function actually uses it, having a `allow_point=False` and doing something like `use_dict = _accuracy_dict.copy()` then `if not allow_point: del _accuracy_dict['point']` or whatever is fine

You might end up wrapping this in a `_get_accuracy_dict(allow_point=False)` for example so you can reuse code